Formula/chapel.rb

```diff
@@ -60,8 +60,14 @@ def install
 
   test do
     ENV["CHPL_HOME"] = libexec
+    ENV["CHPL_GMP"] = "system"
+    ENV["CHPL_INCLUDE_PATH"] = HOMEBREW_PREFIX/"include"
+    ENV["CHPL_LIB_PATH"] = HOMEBREW_PREFIX/"lib"
     cd libexec do
       system "util/test/checkChplInstall"
+      with_env(CHPL_LLVM: "none") do
+        system "util/test/checkChplInstall"
+      end
```

Comment:
Can we repeat this also with `CHPL_TARGET_COMPILER: "system"` to make sure we really are getting an LLVM build and that the LLVM backend works?